src/core/Core_profiling.ml

```diff
@@ -133,6 +152,14 @@ type t = {
 }
 [@@deriving show]
 
+let merge a b : t =
```

Comment:
could be good to explain why we need such a function. My guess it's needed by semgrep-pro, but then would be good to give the context why we need it, because it's not called at all in the semgrep codebase, so someone could just remove it by mistake.

Reply:
done